utils/dataparser.py

```diff
@@ -0,0 +1,29 @@
+import base64
+import io
+import pandas as pd
+
+HYPO_EXT = '.hypo'
+
+
+def parse_contents(contents, filename):
+    """
+    Parse the input into a dataframe using correct parser
+    depending on the file extension
+    """
```

Comment:
Same as above, maybe add argument documentation?

Reply:
Also, should we document what this function returns?

Reply:
Here changed the documentation to reflect the returned value and added info about the arguments